docs/features/deploying/manage-projects.md

```diff
@@ -2,6 +2,40 @@
 title: Manage Projects
 ---
 
+## Hostname / Alias
+
+The **Alias** feature gives the ability to add the same KintoBlock more than once in a single project by changing the hostname. The changes made to the original KintoBlock will be applied to all the aliases.
+
+To add the same KintoBlock multiple times in a Project, follow the instructions below:
+
+1. Select **Projects** from the sidebar.
+
+2. Click **Create New Project** card or the **+** icon.
+
+3. In the **Basic Info** form, specify details on `Project Name`, `Internal Name` and `Environment Name`.
+
+4. Under the KintoBlocks section, search for the Kintoblock you would like to add. E.g. Microservice Example.
+
+5. Again search for the same KintoBlock and add it to the Project.
```

Comment:
Make a separate document for "Searching again and adding multiple kinto blocks".